lessons/pydata/visualization_basics/index.ipynb

```diff
@@ -0,0 +1,612 @@
+{
```

Comment:
<p>Někam bych dal odkaz na https://www.data-to-viz.com/</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Přidávám do příslušné issue.</p>